licenses/mit_license.html

```diff
@@ -0,0 +1,372 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head profile="http://www.w3.org/1999/xhtml/vocab">
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
+<link rel="canonical" href="/licenses/MIT" />
+<link rel="shortlink" href="/node/66" />
+<meta name="MobileOptimized" content="width" />
+<meta name="HandheldFriendly" content="true" />
+<meta name="viewport" content="width=device-width, initial-scale=1" />
+<link rel="shortcut icon" href="https://opensource.org/files/osi_favicon.png" type="image/png" />
+  <title>The MIT License | Open Source Initiative</title>
+  <link type="text/css" rel="stylesheet" href="https://opensource.org/files/css/css_xE-rWrJf-fncB6ztZfd2huxqgxu4WO-qwma6Xer30m4.css" media="all" />
```

Comment:
A bit worried about the external .css links here ...
Can the .css be copied into our own repository rather than calling out to a potentially vulnerable site?

Reply:
as above